cocos/render-scene/scene/camera.ts

```diff
@@ -996,6 +996,12 @@ export class Camera {
         if (!this._node) return;
 
         let viewProjDirty = false;
+        const xr = globalThis.__globalXR;
+        if (xr.isWebXR && xr.webXRWindowMap && xr.webXRMatProjs) {
```

Comment:
xr may be null or undefined?

Reply:
In game.ts, init() will execute initXR() and initialize globalThis.__globalXR is an object, which can be understood as a runtime rt object or global window variable. I don't think it's necessary to judge everything as null.

Reply:
game.ts

Reply:
game.ts private _initXR () {
        if (typeof globalThis.__globalXR === 'undefined') {
            globalThis.__globalXR = {};
        }...